### PR TITLE
Adding google scholar field and 2 other

### PR DESCRIPTION
**Pull request by Suryansh5545**
#### Changes proposed in this pull request:- Added Google scholar and other field in profile page

-added 3 input field for google scholar , linkedin , github. 

-
evalai-ngx profile

### CODE REVIEW

File: src/app/components/profile/profile.component.html [hunk 1]
```diff
@@ -38,6 +38,29 @@
...
+				</div>
+			</div>
+				<div class="item">
+					<span class="text-light-black fw-semibold fs-12">
+						GitHub Url<br>
+					</span>
+					<div class="content fw-regular">
+						{{user.github_url}}
+					</div>
+			</div>
```
Comment (lunayach) [inline src/app/components/profile/profile.component.html:55]: Indentation is off here.

File: src/app/components/profile/profile.component.html [hunk 2]
```diff
@@ -38,6 +38,29 @@
...
+			<div class="item">
```
Comment (lunayach) [inline src/app/components/profile/profile.component.html:56]: Also, here.

File: src/app/components/profile/profile.component.html [hunk 3]
```diff
@@ -59,7 +82,7 @@
 				</div>
 
 				<div class="modal-container" [class.hidden]="!isTokenModalVisible">
-					<div class="modal-card">
+					<div class="modal-card" >
```
Comment (lunayach) [inline src/app/components/profile/profile.component.html:84]: Please remove these unnecessary spaces.

**Comment by Suryansh5545** [comment]
I fix the indent issue and removed unnecessary lines

**Comment by Suryansh5545** [comment]
i am opening new pr

**Pull request closed without merging**